### DISCUSSION TITLE
리더보드 관련해서 질문입니다.

### DISCUSSION THREAD

**Discussion by jiwoo50** [naverz/zepeto-studio-kor#968] [5. Q&A]
LeaderboardAPI.SetScore 를 사용해서 사용자의 점수를 등록하고 나서 갱신된 리더보드를 유지해야하는데 여기서 리더보드의 랭크 정보들을 서버에 따로 저장을 해서 관리를 해야하나요? 아니면 LeaderboardAPI만으로도 저장및 관리가 되나요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, LeaderboardAPI 만으로도 저장 및 관리가 됩니다.
다만 특정 주기마다 초기화 되도록 설정하셨다면, 이전 기간 자료는 초기화 되므로 
영구적으로 보관해야 하는 자료들은 Datastorage를 통해서 따로 저장하는 것도 방법입니다.

**Reply by Sonak0930** [reply]
그런데 LeaderBoardAPI로 저장하는건 로컬 본인 점수만 저장되는 거 같던데, 다른 유저들의 스코어 정보도 한 리더보드에 같이 저장할 수 없을까요?
